sys/platform/src/lib.rs

```diff
@@ -0,0 +1,199 @@
+#![no_std]
```

Comment:
Please add some doc comments to the crate to describe what it's doing.
Same for each module.

Reply:
I added header doc for each src module in platform crate. Let me know if there's room for improvement.